src/main/java/redis/clients/jedis/commands/BinaryJedisCommands.java

```diff
@@ -358,4 +358,6 @@
   List<byte[]> xclaim(byte[] key, byte[] groupname, byte[] consumername, long minIdleTime, long newIdleTime, int retries, boolean force, byte[][] ids);
 
   Object sendCommand(ProtocolCommand cmd, byte[]... args);
```

Comment:
I missed this earlier. I think, we shouldn't include these here. These interfaces imitates actual Redis commands. But `sendCommand` is not a Redis command. I.e. there isn't any `SEND COMMAND` command. We may have to come up with some other solutions for this.

Reply:
@sazzad16 a proposal: I'll remove the sendCommand entries from all affected interfaces. The definition and implementation will be only in the classes themselves.

Reply:
@DvirDukhan I'm okay with that, specially in beginning. We may need a new interface for this.

@gkorland thoughts?

Reply:
@sazzad16 I will open an issue of the new interface(s) required, so we can discuss it there. Meanwhile, I will remove the additional commands from the interface